src/library/Registrar/Adapter/Resellerclub.php

```diff
@@ -128,33 +128,25 @@ public function modifyNs(Registrar_Domain $domain)
 
     public function modifyContact(Registrar_Domain $domain)
     {
-        $customer = $this->_getCustomerDetails($domain);
-        $cdetails = $this->_getDefaultContactDetails($domain, $customer['customerid']);
-        $contact_id = $cdetails['Contact']['registrant'];
-
         $c = $domain->getContactRegistrar();
-
-        $required_params = [
-            'contact-id' => $contact_id,
-            'name' => $c->getName(),
-            'company' => $c->getCompany(),
-            'email' => $c->getEmail(),
-            'address-line-1' => $c->getAddress1(),
-            'city' => $c->getCity(),
-            'zipcode' => $c->getZip(),
-            'phone-cc' => $c->getTelCc(),
-            'phone' => $c->getTel(),
-            'country' => $c->getCountry(),
-        ];
-
-        $optional_params = [
-            'address-line-2' => $c->getAddress2(),
-            'address-line-3' => $c->getAddress3(),
-            'state' => $c->getState(),
+        $old_contact = $this -> getDomainDetails($domain);
+        /*
+        Check if the email address is the same as the one in the domain details
+        Current Fossbilling API uses the email adress of the registrant as "Username" for reseller club account
+        */
+        if($old_contact->getContactAdmin()->getEmail() == $c->getEmail()){
```

Comment:
getting an "Unable to change email address (9999)" when I leave the email address untouched. (only changing the house number of the address.

I think this here should be:
```suggestion
        if($old_contact->getContactAdmin()->getEmail() != $c->getEmail()){
```

Please correct me if i'm wrong.